_search-plugins/ubi/ubi-dashboard-tutorial.md

```diff
@@ -0,0 +1,98 @@
+has_children: false
+nav_order: 7
+---
+
+
+# Build an analytic dashboard for UBI
+
+Whether you've been collecting user events and queries for a while, or [you uploaded some sample events](https://github.com/o19s/chorus-OpenSearch-edition/blob/main/katas/003_import_preexisting_event_data.md), now you're ready to visualize them in an OpenSearch Dashboards dashboard using User Behavior Insights (UBI).
+
+To view the resulting dashboard without following the tutorial, follow these steps:
+1. Download and save the [sample UBI dashboard]({{site.url}}{{site.baseurl}}/assets/examples/ubi-dashboard.ndjson).
+1. On the top menu, go to **Management > Dashboard Management**.
+1. From the **Dashboards** panel, choose **Saved objects**.
+1. In the upper right corner, select **Import**.
+1. In the **Select file** panel, choose **Import**.
+1. Select the UBI dashboard file that you downloaded and select the **Import** button.
+
+## 1. Start OpenSearch Dashboards
+
+Depending on your configuration: `http://{server}:5601/app/home#/`
+![Dashboard Home]({{site.url}}{{site.baseurl}}/images/ubi/home.png "Dashboards")
+{: .note} 
+
+## 2. Create an index pattern
+
+Within OpenSearch Management navigate to **Dashboards Management > Index patterns**.
+
+`http://{server}:5601/app/management/OpenSearch-dashboards/indexPatterns`
+
+It will look something like the following.
+
+OpenSearch Dashboards access your indexes using index patterns. To visualize your users' online search behavior, you must create an index pattern in order to access the indexes that UBI creates. For more information, see [Index patterns]({{site.url}}{{site.baseurl}}/dashboards/management/index-patterns/).
+
+After you select **Create index pattern**, you'll see a list of indexes in your OpenSearch instance. The UBI stores may be hidden by default, so be sure to select **Include system and hidden indexes**, as shown in the following image. 
+![Index Patterns]({{site.url}}{{site.baseurl}}/images/ubi/index_pattern2.png "Index Patterns")
+
+You can group indexes into the same data source for your dashboard using wildcards. For this tutorial we'll combine the query and event stores into the `ubi_*` pattern.
+
+OpenSearch Dashboards prompts you to filter on any `date` field in your schema, so that you can look at things like trending queries over the last 15 minutes. However, for your first dashboard, select **I don't want to use the time filter**, as shown in the following image. 
+<img src="{{site.url}}{{site.baseurl}}/images/ubi/index_pattern3.png" alt="Index Patterns" width="400"/>
+
+
+After selecting **Create index pattern, you're ready to start building a dashboard that displays the UBI store data.
+
+## 3. Create a new dashboard
+
+To create a new dashboard, on the top menu go to **OpenSearch Dashboards > Dashboards**, then select **Create > Dashboard** and choose **Create new**.
+If you haven't previously created a dashboard, you should see something like the following:
```

Comment:
```suggestion
If you haven't previously created a dashboard, you are presented with the option to create a new dashboard. Otherwise, previously created dashboards are displayed.
```